pkg/utils/common/common.go

```diff
@@ -344,6 +349,42 @@ func askToChooseOneResource(app *v1beta1.Application, filters ...clusterObjectRe
 	return nil, fmt.Errorf("choosing resource err %w", err)
 }
 
+// AskToChooseOneNamespace ask for choose one namespace as env
+func AskToChooseOneNamespace(c client.Client, envMeta *types.EnvMeta) error {
+	var nsList v1.NamespaceList
+	if err := c.List(context.TODO(), &nsList); err != nil {
+		return err
+	}
+	var ops = []string{CreateCustomNamespace}
+	for _, r := range nsList.Items {
+		ops = append(ops, r.Name)
+	}
+	prompt := &survey.Select{
+		Message: "Would you like to choose an existing namespaces as your env?",
+		Options: ops,
+	}
```

Comment:
Please use a Confirm instead of asking user to input long text like "create new namespace".
```go
var useExist bool
prompt := &survey.Confirm{
    Message: "Would you like to choose an existing namespace as your env?",
}
survey.AskOne(prompt, &useExist)
if useExist{...}
```